jenkins/test

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -l
+
+set -e
+
+if [ -f /.dockerenv ]; then
+    source ~/.bashrc
+    rvm use default
+    mv spec/dummy/config/database{-jenkins,}.yml
+fi
+
+export RAILS_ENV=test
+export BUNDLE_WITHOUT=development:build
+
+exit_code=0
+
+function run {
+    declare -a tests_command=("$@")
+
+    echo ''
+    echo "=== Running \`${tests_command[*]}\`"
+    if ! ${tests_command[*]}; then
+        echo "=== These tests failed."
+        exit_code=1
+    fi
+}
+
+function info {
+    declare -a info_command=("$@")
+    echo ''
+    echo "=== Running for informational purposes \`${info_command[*]}\`"
+    if ! ${info_command[*]}; then
+        echo "== This test has errors and/or warnings. Please review results"
+    fi
+}
+
+run mkdir -p spec/dummy/tmp/capybara
+run bundle install --quiet
+run bundle update brakeman --quiet
+run bundle exec rake db:migrate
```

Comment:
Going to switch to `db:setup` and check

Reply:
I would suggest perhaps `db:create db:schema:load db:migrate` explicitly. Rationale:

* More explicit, immediately clear what's going on without having to remember what setup actually does.
*  `db:setup` also adds seeds. 
    * Seeds aren't used or written consistently across the projects - I recall a recent example where running the seeds task in the test env would break the test run.
    * I think generally it's a good idea not to rely on or account for seeds in a test suite anyway.

Although this project doesn't actually have a db/seeds file, using setup here would set a sub-optimal precedent, IMHO.

Reply:
Makes sense, moved to the `create, schema:load, migrate` approach.